modules/beacon/mkosi.extra/usr/local/lib/systemd/system/publish-certificate.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Publish SEV certificate using beacon
+DefaultDependencies=no
+After=sev-certificate-generator.service
+Wants=sev-certificate-generator.service
+Requires=enable-beacon.service
```

Comment:
We probably want Requires/After for beacon-boot.service here. Also, we should probably add the avahi requirement too.